hs_communities/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+
+class Topic(models.Model):
+    name = models.CharField(editable=True, null=False, max_length=255)
+
+    # order = models.IntegerField(null=False,
+    #                             default=0,
+    #                             editable=True,
+    #                             help_text='Position of this entry: 1-n')
+
+    def __str__(self):
+        return "{}".format(self.name)
+
+    @classmethod
+    def add(cls, new_topic):
+        """
+        Add a new topic
+
+        :param new_topic: string topic to be added
+        :return:
+        """
+        topic = cls()
+        topic.name = new_topic
+        # topics_list = Topic.objects.all.values_list("order", flat=True)
+        # topic.order = max(topics_list if topics_list else [0]) + 1
+
+        topic.save()
```

Comment:
Is a ```update``` class method also needed here for topic editing?

Reply:
Have a look at hs_communities/static/js/topics.js `saveEdit` and see if that addresses your question

Reply:
@cuahsimarko I took a look at it and see topic saveEdit() AJAX just calls TopicsView to process post requests to add or update a topic. That TopicsView class just directly create or update a Topic object without using this add class method. I don't know enough about how Topics model is being used, so cannot give any specific comment on the current implementation. As long as @sblack-usu approves this PR, I am fine with it. It'd be definitely helpful to see a demo on the functionality before doing code review, though.

Reply:
@hyi The demo for this was delayed by some organizational factors. I will be showing the UI on a Wednesday HydroShare call as soon as possible, a couple things need to happen first. I'll see how long I can keep this PR open without having to continually merge dev and deconflict. As long as that's easy we can wait. Otherwise, I'll ask later to get this in and make minor changes in a subsequent PR.

Reply:
@cuahsimarko Sounds good.